### DISCUSSION TITLE
2022/04/Leanote-Linux-app-startup/

### DISCUSSION THREAD

**Discussion by giscus** [Li-Aaron/Li-Aaron.github.io#4] [Announcements]
# 2022/04/Leanote-Linux-app-startup/

本文介绍之前没有解决的Leanote APP在Linux没有图标的问题。这里重新介绍一下添加启动器和图标的方法。

https://li-aaron.github.io/2022/04/Leanote-Linux-app-startup/

**Comment by ac2sherry** [comment]
再测试一下
